src/dodal/devices/vmxm/vmxm_attenuator.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from ophyd import Component, Device, EpicsSignal, Kind
+from ophyd.status import Status, SubscriptionStatus
+
+from dodal.devices.detector import DetectorParams
+from dodal.devices.status import await_value
+from dodal.log import LOGGER
+
+
+class VmxmAttenuator(Device):
```

Comment:
It would be nice to have an `Attenuator` _interface_ - the details of this class are quite different to i03 but the concept is the same. Might be out of scope for this issue though - write up separate issue.

Reply:
See https://github.com/DiamondLightSource/hyperion/issues/956